Author: Okinea Dev <hi@okinea.dev>
Description: Removed suspicious default permissions for audio recording and default permissions for network providers that the user doesn't even know about

diff --git a/images/super/product_a/etc/default-permissions/default-permissions-dtag.xml b/images/super/product_a/etc/default-permissions/default-permissions-dtag.xml
deleted file mode 100644
index 0000000..0000000
--- a/images/super/product_a/etc/default-permissions/default-permissions-dtag.xml
+++ /dev/null
@@ -1,10 +0,0 @@
-<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
-<exceptions>
-    <exception package="com.aura.oobe.deutsche">
-        <permission name="android.permission.POST_NOTIFICATIONS" whitelisted="true" />
-    </exception>
-    <exception package="de.telekom.tsc">
-        <permission name="android.permission.READ_PHONE_STATE" fixed="false" />
-        <permission name="android.permission.POST_NOTIFICATIONS" whitelisted="true" />
-    </exception>
-</exceptions>
\ No newline at end of file
diff --git a/images/super/product_a/etc/default-permissions/default-permissions-google.xml b/images/super/product_a/etc/default-permissions/default-permissions-google.xml
index 0000000..0000000 100644
--- a/images/super/product_a/etc/default-permissions/default-permissions-google.xml
+++ b/images/super/product_a/etc/default-permissions/default-permissions-google.xml
@@ -16,20 +16,6 @@
     <permission name="android.permission.POST_NOTIFICATIONS" fixed="false"/>
   </exception>
 
-  <exception
-    package="com.google.android.apps.assistant"
-    sha256-cert-digest="43:0B:20:EB:61:96:7E:9C:5B:BE:02:59:9B:07:35:16:80:7D:AE:72:D7:86:FE:92:7F:48:96:B7:C7:F2:96:1F">
-    <!-- Record audio -->
-    <permission name="android.permission.RECORD_AUDIO" fixed="false"/>
-  </exception>
-
-  <exception
-    package="com.google.android.apps.actionsservice"
-    sha256-cert-digest="43:0B:20:EB:61:96:7E:9C:5B:BE:02:59:9B:07:35:16:80:7D:AE:72:D7:86:FE:92:7F:48:96:B7:C7:F2:96:1F">
-    <!-- Record audio -->
-    <permission name="android.permission.RECORD_AUDIO" fixed="false"/>
-  </exception>
-
   <exception
     package="com.google.android.marvin.talkback"
     sha256-cert-digest="7B:5A:D5:51:80:A4:8A:1F:30:F3:53:77:C0:F9:E5:F9:11:BF:94:2F:B4:CF:83:EB:A2:55:A0:EB:F5:80:BE:EF">
diff --git a/images/super/product_a/pangu/system/etc/permissions/facebook-hiddenapi-package-whitelist.xml b/images/super/product_a/pangu/system/etc/permissions/facebook-hiddenapi-package-whitelist.xml
deleted file mode 100644
index 0000000..0000000
--- a/images/super/product_a/pangu/system/etc/permissions/facebook-hiddenapi-package-whitelist.xml
+++ /dev/null
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<config>
-        <hidden-api-whitelisted-app package="com.facebook.appmanager" />
-        <hidden-api-whitelisted-app package="com.facebook.system" />
-        <hidden-api-whitelisted-app package="com.facebook.services" />
-</config>
\ No newline at end of file
diff --git a/images/super/product_a/pangu/system/etc/permissions/facebook-miui.xml b/images/super/product_a/pangu/system/etc/permissions/facebook-miui.xml
deleted file mode 100644
index 0000000..0000000
--- a/images/super/product_a/pangu/system/etc/permissions/facebook-miui.xml
+++ /dev/null
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<permissions>
-    <allow-in-power-save-except-idle package="com.facebook.appmanager" />
-</permissions>
diff --git a/images/super/product_a/pangu/system/etc/permissions/privapp-permissions-dtag.xml b/images/super/product_a/pangu/system/etc/permissions/privapp-permissions-dtag.xml
deleted file mode 100644
index 0000000..0000000
--- a/images/super/product_a/pangu/system/etc/permissions/privapp-permissions-dtag.xml
+++ /dev/null
@@ -1,19 +0,0 @@
-<permissions>
-
-    <privapp-permissions package="de.telekom.tsc">
-        <permission name="android.permission.INSTALL_PACKAGES" />
-        <permission name="android.permission.READ_PRIVILEGED_PHONE_STATE" />
-        <permission name="android.permission.PACKAGE_VERIFICATION_AGENT"/>
-        <permission name="android.permission.START_ACTIVITIES_FROM_BACKGROUND"/>
-    </privapp-permissions>
-
-    <privapp-permissions package="com.aura.oobe.deutsche">
-        <permission name="android.permission.INSTALL_PACKAGES"/>
-    </privapp-permissions>
-
-    <allow-in-power-save package="de.telekom.tsc"/>
-
-    <allow-in-power-save-except-idle package="de.telekom.tsc"/>
-
-    <allow-in-power-save package="com.aura.oobe.deutsche"/>
-</permissions>
\ No newline at end of file
